### DISCUSSION TITLE
Convert supercell format FC3 to primitive cell format FC3

### DISCUSSION THREAD

**Discussion by rfhari** [ttadano/alamode#226] [Q&A] [answered]
Hello, I'm relatively new to using phonon property calculation software like Phono3py and Alamode. 

I'm currently comparing the second- and third-order force constants (FC2 and FC3) calculated using Phono3py with those from Alamode for AlN. It appears that Alamode calculates FC3 in a compact format, i.e., (num_patom, num_satom, num_satom), whereas Phono3py calculates it in a full format, i.e., (num_satom, num_satom, num_satom). Could you please let me know if there is any functionality (or a mapping script) iavailable within Alamode to convert Phono3py's?

**Comment by ttadano** [comment, marked as answer]
Hi,

We do not have a script that converts Alamode format to the Phono3py format. So, please write a script by yourself.

The discussion and sample code at #192 may be helpful.
